app/parsers/bulkrax/application_parser.rb

```diff
@@ -75,10 +75,9 @@ def related_children_parsed_mapping
     def get_field_mapping_hash_for(key)
       return instance_variable_get("@#{key}_hash") if instance_variable_get("@#{key}_hash").present?
 
-      mapping = importerexporter.field_mapping == [{}] ? {} : importerexporter.field_mapping
       instance_variable_set(
         "@#{key}_hash",
-        mapping&.with_indifferent_access&.select { |_, h| h.key?(key) }
```

Comment:
because `Bulkrax::Exporter.find(2).field_mapping` was returning `nil`, line 81 returned `nil`. this resulted in an error on line 83 because `instance_variable_get("@#{key}_hash")` evaluated to `nil` and `nil.length` throws an error.